incubator/pom.xml

```diff
@@ -28,6 +28,9 @@
     <module>command-log</module>
     <module>command-dump</module>
     <module>command-tune</module>
+
+    <module>exporter-otlp</module>
+    <module>exporter-otlp.spec</module>
```

Comment:
All specs go first, later they will all move to `specs` project.

Reply:
done